README.md

```diff
@@ -112,6 +123,11 @@ Validation errors occurred importing the spreadsheet.
 
 Then the indicated role, here `caseworker-cmc-loa1`, must be added to CCD (See [2. Add roles](#2-add-roles)).
 
+### NOTE: 
+The user and role creation can be done by executing the script
+
+ccd-docker$ ./bin/create_roles.sh
```

Comment:
please update the name of the script.

Reply:
We are doing both initial users and roles. So, why the script having name initial roles, only?
How about "create-initial-roles-and-users"?

Reply:
Renamed the file.